seqr/views/apis/individual_api_tests.py

```diff
@@ -371,6 +372,20 @@ def test_individuals_table_handler(self):
         self.assertEqual(len(errors), 1)
         self.assertEqual(errors[0], 'Missing required columns: Individual Id')
 
+        response = self.client.post(individuals_url, {'f': SimpleUploadedFile(
+            'test.tsv', 'Family ID	Individual ID\n""	""""'.encode('utf-8'))})
+        self.assertEqual(response.status_code, 400)
+        self.assertDictEqual(response.json(), {'warnings': None, 'errors': [
+            'Missing Family Id in row #1', 'Missing Individual Id in row #1',
+        ]})
+
+        response = self.client.post(individuals_url, {'f': SimpleUploadedFile(
+            'test.tsv',  '#Some comments\n#Family ID	#Individual ID	Previous Individual ID\n"1"	"NA19675_1"""'.encode('utf-8'))})
+        self.assertEqual(response.status_code, 400)
+        self.assertDictEqual(response.json(), {'warnings': [], 'errors': [
+            'Error while parsing file: test.tsv. Row 1 contains 2 columns: 1, NA19675_1, while header contains 3: Family ID, Individual ID, Previous Individual ID',
```

Comment:
The test data `#Some comments\n#Family ID	#Individual ID	Previous Individual ID\n` is weird. The header line starts with a `#` and looks like a comment line.

Reply:
This is maintaining existing behavior and moves an [existing test](https://github.com/broadinstitute/seqr/blob/master/seqr/views/utils/pedigree_info_utils_tests.py#L71), as we do allow `#` characters to be ignored in file headers. . Changing supported behavior is out of scope for this PR